### PR TITLE
Respond to keyboard for Messages view

### PR DESCRIPTION
**Pull request by justinvdm**
We need to shift the height of the view when the keyboard appears/disappears.

### CODE REVIEW

**Comment by justinvdm** [comment]
Depends on #289, otherwise ready for review.

File: src/api/__tests__/parse-test.js [hunk 1]
```diff
@@ -53,13 +53,6 @@ describe('api/parse', () => {
     });
   });
 
-  describe('parseActivities', () => {
-    it('should parse each activity', () => {
-      expect(parseActivities([fakeActivity()]))
-        .toEqual([parseActivity(fakeActivity())]);
```
Comment (codiebeulaine) [inline src/api/__tests__/parse-test.js]: @justinvdm how come we are taking this out?

**Comment by codiebeulaine** [comment]
👍

**Comment by justinvdm** [comment]
@codiebeulaine We removed `parseActivities`, since
`parseActivityListResults` doesn't need it anymore:
https://github.com/praekeltfoundation/mentorship-droid/pull/290/files#diff-e7a6114e2230671a37fef87a5a79560aR82

**Pull request merged**